alacritty_terminal/src/grid/storage.rs

```diff
@@ -310,8 +310,21 @@ impl<T> IndexMut<Line> for Storage<T> {
 mod test {
     use crate::grid::row::Row;
     use crate::grid::storage::Storage;
+    use crate::grid::GridCell;
     use crate::index::{Column, Line};
 
+    impl GridCell for char {
+        fn is_empty(&self) -> bool {
+            *self == ' ' || *self == '\t'
```

Comment:
Are these really the only two "empty" characters?

Reply:
Yes. Generally this isn't about empty text characters, but more about what Alacritty treats as empty characters. By default all characters are filled with `' '` if they're not filled with anything, so that's basically the most important thing that needs to be checked.

The tabs are basically just an addition to that. Since we set those ourselves with the tabstop logic, we can treat them as empty too, since often with ncurses they are just used to move the cursor around more efficiently.

Everything else are basically characters which come in from the PTY. These are explicitly set so we don't really want to treat them as empty. The method is more about finding cells which are unchanged, rather than cells which don't need to be rendered.

Reply:
Would a different name actually make more sense here then. Like `is_spacer` since a "spacer" sounds more like something we've added ourselves.

Reply:
No, since we also want to check other things like foreground color. This is basically just to check if a cell can be ignored or assumed to be default, so `empty` is good here I think.